src/components/tabs/OnboardLoggingTab.vue

```
@@ -428,22 +462,45 @@ export default defineComponent({
+                let loggingFrequency = loggingFrequencyHz;
                 let loggingFrequencyUnit = "Hz";
                 if (gcd(loggingFrequency, 1000) === 1000) {
                     loggingFrequency /= 1000;
                     loggingFrequencyUnit = "kHz";
                 }
+                const overCap = ringActive && loggingFrequencyHz > RING_MAX_FRAME_HZ;
                 rates.push({
                     value: i,
-                    label: `1/${Math.pow(2, i)} (${loggingFrequency}${loggingFrequencyUnit})`,
+                    label: `1/${Math.pow(2, i)} (${loggingFrequency}${loggingFrequencyUnit})${
+                        overCap ? " — " + i18n.getMessage("onboardLoggingRateRingExceedsCap") : ""
```

ESLint

[error] Unexpected string concatenation.

(prefer-template)